openghg/standardise/emissions/_intem.py

```diff
@@ -0,0 +1,100 @@
+from pathlib import Path
+from typing import Dict, Literal, Optional, Union
+
+
+def parse_intem(
+    filepath: Path,
+    species: str,
+    source: str = "intem",
+    domain: str = "europe",
+    data_type: str = "emissions",
+    database: Optional[str] = None,
+    database_version: Optional[str] = None,
+    model: Optional[str] = None,
+    period: Optional[Union[str, tuple]] = None,
+    chunks: Union[int, Dict, Literal["auto"], None] = None,
+    continuous: bool = True,
+) -> Dict:
+    """
+    Parse INTEM emissions data from the specified file.
+
+    Args:
+        filepath (Path): Path to the '.nc' file containing INTEM emissions data.
+        species (str): Name of species
+        source (str): Source of the emissions data, default is 'intem'.
+        domain (str): Geographic domain, default is 'europe'.
+        data_type (str): Type of data, default is 'emissions'.
+        database (Optional[str]): Database name if applicable.
+        database_version (Optional[str]): Version of the database if applicable.
+        model (Optional[str]): Model name if applicable.
+        period : "needs explainer"
+        chunks : (Union[int, Dict, Literal["auto"], None]): Chunking configuration.
+        continuous (bool): "needs explainer"
```

Comment:
```suggestion
        continuous: Whether time stamps have to be continuous.
```

Reply:
@rt17603 can you please advice on this?

Reply:
With emissions/footprints/boundary conditions data in our standard format we often have to infer a time period associated with the time points. The "continuous" constraint comes in within the `openghg.store._infer_time.infer_date_range()` function where this is checking whether it has been possible to infer this detail e.g. https://github.com/openghg/openghg/blob/devel/openghg/store/_infer_time.py#L138.

If this is set to False this carries on without this piece of information (though I believe Gareth found a place this falls down https://github.com/openghg/openghg/issues/623 - the specific case got fixed but fixing the additional corner cases may have been deferred until they become clear). If this is set to True this raises a ValueError and exits.